src/main/java/gov/usds/case_issues/controllers/UserDetailsApiController.java

```diff
@@ -2,34 +2,37 @@
 
 import java.util.HashMap;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Profile;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.security.core.Authentication;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import gov.usds.case_issues.db.model.OAuthUser;
+import gov.usds.case_issues.db.repositories.UserRepository;
+
 /**
  * Controller to see information about the current user.
  */
 @RestController
 @Profile("auth-testing")
 public class UserDetailsApiController {
 
+	@Autowired
+	private UserRepository _userRepo;
```

Comment:
We probably shouldn't have repositories direct in the controller layer—not sure it matters all that much, but weird transaction-management things can happen when you're calling directly instead of wrapping them in a service object (also, the fewer direct users the repo has the better, though obviously that's a somewhat illusory benefit in this case.)